README.md

```diff
@@ -1 +1,3 @@
-# external-visio-connector
\ No newline at end of file
+# external-visio-connector
+
+| `webconferencing.externalVisio.active`                            | true     | Indicates if the ExternalVisio connector is active.       |
```

Comment:
Can you a small description ? Explaining what is this property, and a header for the table